api/auth/oktaAuth.test.js

```diff
@@ -40,101 +39,6 @@ tap.test('okta wrappers', async oktaTests => {
     }
   );
 
-  oktaTests.test('call Okta endpoint', async callOktaEndpointTest => {
```

Comment:
we weren't actually using the callOktaEndpoint method anywhere. I think I was added back when we thought we were going to interact with Okta more. I removed it because it wasn't being used.